config/samples/ccruntime/peer-pods/kustomization.yaml

```diff
@@ -10,13 +10,19 @@ images:
 - name: quay.io/confidential-containers/container-engine-for-cc-payload
   newTag: c80c3e7392c1562841e75072d227e07f98cab2bd 
 - name: quay.io/confidential-containers/runtime-payload
-  newName: quay.io/confidential-containers/runtime-payload-ci
```

Comment:
Unintentional I guess ?

Reply:
There's one `do-not-merge` commit (the last one, it's described as do-not-merge) that allows me to test with one PR that I need to get merged on the Kata Containers side.